src/lib.rs

```diff
@@ -246,7 +246,7 @@ where
     let inputs_secondary: NovaAugmentedCircuitInputs<G1> = NovaAugmentedCircuitInputs::new(
       pp.digest(),
       G2::Scalar::ZERO,
-      z0_secondary,
+      z0_secondary.to_vec(),
```

Comment:
This [clones](https://doc.rust-lang.org/std/primitive.slice.html#method.to_vec) the vector.